src/pages/ProjectPage/ProjectPage.tsx

```diff
@@ -0,0 +1,85 @@
+import projectData from "../../data/dummy_data_extended.json";
+import { useParams } from "react-router-dom";
+import { Link } from "react-router-dom";
+
+function ProjectPage() {
+  const { id } = useParams();
+  const jobDetails = projectData.jobs.find(
+    (job) => job.id === parseInt(id, 10)
```

Comment:
you have to parse int? isn't the id coming in a number?

Reply:
It is coming in the string form and needs to be parsed